packages/components/ovh-shell/features/client/shell-client.feature

```diff
@@ -6,3 +6,7 @@ Scenario: Plugin method invokation
   When I invoke a method from my plugin
   Then The invokation should be resolved with the method result
 
+Scenario: Plugin event listener
+  When My shell and shell client are configured with a direct message bus
```

Comment:
`When` should be used when an action happens. This line is more related to configuration/prerequisites, so it should be `Given`. 

Also, to respect the Gherkin pattern, we should always have `Given, When, Then` and they are nearly the same as the AAA pattern ( Arrange, Act, Assert ). 

In this case, it should be more like : 

`Given My shell and shell client are configured with a direct message bus`
`And I add an event listener on the shell client`
`When The shell emits an event`
`Then The event emitted from the shell should be received by the client`